utils/update-consul-conf

```diff
@@ -288,6 +293,13 @@ fi
 SVCS_CONF=''
 
 hax_http_port=$(get_hax_http_port_from_kv_file)
+hax_ssl_config=$(get_hax_ssl_config)
+hax_http_protocol=$(jq -r ".http_protocol" <<<$hax_ssl_config)
+if [[ "hax_http_protocol" != "https" ]]; then
+    hax_http_protocol="http"
+fi
+hax_ssl_cert_path=$(jq -r ".cert_path" <<<$hax_ssl_config)
```

Comment:
@DmitryKuzmenko, is `hctl bootstrap` working? what if `ssl/hax` is not set?

Reply:
With this update it is not working because `ssl/hax` is not set there. It worked correctly when the ssl config has been read from the kv file.